lib/network_utils.pm

```diff
@@ -22,9 +27,13 @@ use mm_network;
 our @EXPORT = qw(setup_static_network recover_network can_upload_logs iface ifc_exists ifc_is_up);
 
 =head2 setup_static_network
+
+ setup_static_network();
```

Comment:
```suggestion
 setup_static_network(ip => '10.0.2.15', gw => '10.0.2.1');
```
parameters are optional so you can skip them , but your version will confuse people because it look like function not accept any parameters at all